drivers/vmwarevcloudair/vcloudair.go

```diff
@@ -407,66 +411,17 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if d.Provision {
-		dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
-
-		log.Infof("Installing Docker...")
-
-		cmd, err = d.GetSSHCommand(dockerInstall)
-		if err != nil {
-			return err
-		}
-
-		if err = cmd.Run(); err != nil {
-			return err
-		}
+	dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
```

Comment:
@tianon if you're referring to the `/ubuntu/` missing, I just omitted it because it's not necessarily ubuntu the target platform (it's the default, but can be passed as a parameter) I did some investigation on the script and figured out it would work on RHEL too, if I'm mistaken (which is likely the case) I'll fix that.